app/Models/Album.php

```diff
@@ -268,4 +274,38 @@ public function newEloquentBuilder($query): AlbumBuilder
 	{
 		return new AlbumBuilder($query);
 	}
+
+	/**
+	 * Set the GPX track for the album.
+	 *
+	 * @param UploadedFile $file the GPX track file to be set
+	 *
+	 * @return void
+	 */
+	public function setTrack(UploadedFile $file): void
+	{
+		if ($this->track_short_path != null) {
```

Comment:
```suggestion
		if ($this->track_short_path !== null) {
```
We're trying to use stronger `!==`/`===` wherever possible.